variables.tf

```diff
@@ -324,6 +324,12 @@ variable "ecr_image_tag_mutability" {
   default = "MUTABLE"
 }
 
+variable "ecr_cross_region_replication_destination_region_names" {
+  description = "If the current region is one of the destination regions, the codepipeline will be triggered when an image with the specified tag is replicated - action-type `REPLICATE`. If the current region is the source region, the action-type of the trigger will be `PUSH`. Image replication needs to be enabled on ECR per region for this to work."
+  type        = list(string)
+  default     = []
+}
+
```

Comment:
Maybe don't hide this functionaliy behind a region but either:

- add `REPLICATE` to the existing `PUSH` action type (both) so the pipeline is started by either action and we don't need a new variable at all, or
- name the variable more explicitly like `pipeline_trigger_action_types` defaulting to `["PUSH"]`